checkstyle-suppressions.xml

```diff
@@ -19,5 +19,5 @@ limitations under the License.
 -->
 <suppressions>
   <!-- Suppress paragraph validation in copyright header -->
-  <suppress files=".*\.java" checks="JavadocParagraph" lines="1-12"/>
+  <suppress files=".*\.java" checks="JavadocParagraph"/>
```

Comment:
I would rather keep the JavadocParagraph check, with the copyright header exclusion, and a separate specific exclusion for the ProjectIdValidator.java.